src/pages/graphql/schema/checkout/queries/get-payment-config.md

```diff
@@ -226,6 +265,7 @@ The `PaymentConfigOutput` contains details about each configured payment method:
 Attribute |  Data Type | Description
 --- | --- | ---
 `ApplePayConfig` | String! | ApplePay payment method configuration
+`GooglePayConfig` | String! | GooglePay payment method configuration
```

Comment:
Aren't these supposed to be "Apple Pay payment method configuration" and "Google Pay payment method configuration" and similar? They ar smashed together here.

Reply:
These are names of output objects.

Reply:
done